src/main/java/io/xpring/xrpl/XRPClientInterface.java

```diff
@@ -0,0 +1,38 @@
+package io.xpring.xrpl;
+
+import io.xpring.xrpl.legacy.LegacyDefaultXRPClient;
```

Comment:
unused import from what I can tell. FYI, IntelliJ has a checkbox on the "Commit Changes" modal to automatically clean up unused imports. It's unchecked by default but I recommend checking it.

![image](https://user-images.githubusercontent.com/4440345/78080643-db876580-736b-11ea-918d-1eb56d5f6e2f.png)

Reply:
Thanks. 

I actually have a TODO to make our linter config actually work (it's committed and it runs but it doesn't auto fix or error, just prints warnings). I'd been mostly deferring these style things until that landed and figured a computer program would just do it for me. 

Cleaned up this one though.